rr_platform/launch/bigoli_platform.launch

```diff
@@ -1,5 +1,6 @@
 <launch>
-    <include file="$(find rr_platform)/launch/bigoli_camera.launch"/>
+    <!--<include file="$(find rr_platform)/launch/bigoli_camera.launch"/>-->
```

Comment:
We should improve our mapper so that we can handle noisy data from the realsense